stable/artifactory-ha/README.md

```diff
@@ -858,6 +858,20 @@ artifactory:
     ## Custom volume comes here ##
 ```
 
+### Custom secrets
+If you need to add a custom secret in a custom init or sidecar container, you can use this option.
+
+For this, there is a section for defining custom secrets in the [values.yaml](values.yaml). By default it's commented out
+```yaml
+artifactory:
```

Comment:
@anastasiagrinman Can you please add this parameter in Configuration section in README  - https://github.com/jfrog/charts/blob/master/stable/artifactory/README.md#configuration

Reply:
Done.

Reply:
artifactory chart will be provided as different PR.